C/factors.C

```diff
@@ -4,12 +4,14 @@ void main()
 {
   int a b;
   pritf("Enter a no:");
-  scan("%d", &a);
+  scanf("%d", &a);
   for(int i=1;i<=a;i++)
   {
     if(a%i=0)
-    b=a/i;
-    printf("%d\n",b)
+      {
```

Comment:
and this file too

Reply:
Updated

Reply:
pls merge whatever i had done...

Reply:
I can only merge when the code quality is good enough